src/test/java/uk/gov/justice/digital/delius/info/PingEndpointTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.delius.info;
+
+import io.restassured.RestAssured;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@RunWith(SpringJUnit4ClassRunner.class)
+@DirtiesContext
+public class PingEndpointTest {
+
+    @LocalServerPort
+    int port;
+
+    @Before
+    public void setup() {
+        RestAssured.port = port;
+        RestAssured.basePath = "/api";
+    }
+
+    @Test
+    public void healthEndpointIsUnauthorised() {
+        String responseBody = given()
+                .when()
+                .get("/ping")
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.OK.value())
```

Comment:
can we check that the content type is plain/text?

Reply:
I spoke to @SimonMitchellMOJ about this and agreed to pick it up in his next PR :grin: